README.md

```diff
@@ -21,7 +21,7 @@ Repository includes four components:
 * **JEasyCryptoServer**, which provides a way to use the Lib from internet, using UDP and JSON.
 * **JEasyCryptoClient**, which uses the Server over network for en-/decryption of text.
 
-Additionally, the code uses the **json-simple** library (see below).
+Additionally, the code uses the **JSON-java** library (see below).
```

Comment:
Could you please add the license of the JSON library?

Reply:
Like adding a link to the License page in the Readme file?

Reply:
Yes good idea. Maybe something like 

> the JSON-java library is licensed under XXX (with clickable link)

Reply:
Thanks for the response. It is done.